solutions/usaco-1232.mdx

```diff
@@ -0,0 +1,101 @@
+---
+id: usaco-1232
+source: USACO Silver 2022 Open
+title: COW Operations
+author: Mythreya Dharani
+---
+## Explanation
+Note that the characters in any queried string can always be moved around in any order.
+For example, OC can be permuted by converting to WCC, and then W, and then finally CO.
+Therefore, the solution might have to do with the parity of the characters. 
+
+Let us define $C$, $O$, and $W$ as the number of respective characters in a substring that is queried. 
+For instance, COWOW would have $C=1$, $O=2$, and $W=2$.
+
+When will it not be possible to convert this substring to a single C? 
+
+We can realize that the parity of $C + W$, $C + O$, or $W + O$ will never change, no matter what operation: 
+
+Removing two adjacent characters does not affect parity.
+Turning one character into the other two will remove the occurence of a character and add the occurence of another character, which again, does not change parity.
+
+Thus, one case when it is not possible is when $O + W$ is odd, as we can never remove all O's and W's.
+Another case is when $O + C$ or $W + C$ is even, since we can never get one C without any O's or W's.
```

Comment:
you could just point them to the sample case & say "look they said it wasn't possible so there's that"
"another way to notice this is..."

Reply:
bump

Reply:
I'll try to do it soon, a little busy rn :)